hvpy/facade.py

```diff
@@ -474,3 +476,49 @@ def downloadMovie(
         hq=hq,
     )
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getNewsFeedInputParameters)
+def getNewsFeed(
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Get the XML RSS feed of the official Helioviewer Project Blog.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getNewsFeed
+    >>> getNewsFeed()
+    '<?xml version="1.0" encoding="utf-8"?><feed xmlns="..." >...</summary></entry></feed>'
+    """
+    params = getNewsFeedInputParameters(
+        callback=callback,
+    )
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(shortenURLInputParameters)
+def shortenURL(
+    queryString: str,
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
```

Comment:
Does it return bytes?

Reply:
It does not but mypy throws an error: 
```
hvpy/facade.py:500: error: Incompatible return value type (got "Union[bytes, str, Dict[str, Any]]", expected "Union[str, Dict[str, Any]]")
hvpy/facade.py:524: error: Incompatible return value type (got "Union[bytes, str, Dict[str, Any]]", expected "Union[str, Dict[str, Any]]")
```

Reply:
I wonder why, something to fix later on.

Reply:
Oh I see why, thats ok.